flarelist2sql.py

```diff
@@ -538,14 +564,18 @@ def main():
         # given_date = ['2023-09-01', '2023-09-10']
         # Iterate through the list of URLs and download files from each
         for url in datfile_urls:
-            file_name_download = download_datfiles_from_url(url, spec_data_dir, timerange)
+            file_name_download = download_datfiles_from_url(url, spec_data_dir, timerange_strp)
```

Comment:
I'm a bit confused about the purpose of this line. It seems like `download_datfiles_from_url` is downloading a file from the specified URL into `spec_data_dir`. However, since the URL (e.g., `http://ovsa.njit.edu/events/yyyy/`) mirrors the contents of `spec_data_dir` (`/common/webplots/events/`), and both are accessible on OVSA, it's unclear why we need to download files from the URL into `spec_data_dir` again. Could you clarify the intended purpose of this step?

Reply:
Yes.` if on_server == 1:` it will not download spec.fits file from URL and find those spec files on `/common/`, `else:`, it will download, in the case that one wants to get the flarelist on their own computer(they can get the flarelist, but will not write any data to SQL).